### ISSUE TITLE
Outdated dependency

### ISSUE THREAD

**Issue by Codename-404** [hoaiduyit/react-pannellum#103]
Do you have plan to update the dependencies of this project? Such as it conflicts with the latest version of react, postcss etc.

**Comment by Dave-Rushabh** [comment]
Yes facing the same issue, is there any tweak to use it with v18.2 of react ?

**Comment by hoaiduyit** [comment]
Hi @Codename-404 @Dave-Rushabh , I'll try to upgrade version to compatible with latest react version.

**Comment by cjauvin** [comment]
We are also using this library, thanks for your great work! We are also being prevented from upgrading to React 18.2 because of it, so we would greatly appreciate an official update.

**Comment by jean-kiepura** [comment]
as suggested by @notnotzero in the following thread (https://github.com/hoaiduyit/react-pannellum/issues/94#issuecomment-1594401224), we fixed our problem by overriding react-pannellum's dependencies in our own package.json. We didn't have any conflict with postcss so we just overrided react and react-dom.

`
 "overrides": {
    "react-pannellum": {
      "react": "$react",
      "react-dom": "$react-dom"
    }
  },
  "dependencies": {
    (.....)
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-pannellum": "^0.2.6",
    (.....)
`
